src/App.config

```diff
@@ -101,6 +101,14 @@
 				<assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
 				<bindingRedirect oldVersion="0.0.0.0-7.0.0.2" newVersion="7.0.0.2" />
 			</dependentAssembly>
+			<dependentAssembly>
+				<assemblyIdentity name="System.Reflection.TypeExtensions" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+				<bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0" />
+			</dependentAssembly>
+			<dependentAssembly>
+				<assemblyIdentity name="PubnubPCL" publicKeyToken="dc66f52ce6619f44" culture="neutral" />
+				<bindingRedirect oldVersion="0.0.0.0-6.19.5.0" newVersion="6.19.5.0" />
```

Comment:
@lotaked , we are using PubnunPCL 6.17.0, wondering why it is redirecting to 6.19.5.

Reply:
Thank you for pointing that out! It was automatically updated by Visual Studio when updating the SDK. I've corrected it now.